### DISCUSSION TITLE
Future Hardware Design Suggestion

### DISCUSSION THREAD

**Discussion by IIAIronWolf** [HASwitchPlate/HASPone#119] [Ideas]
I have noticed that the display backlight is visible underneath the white 3D printed enclosure. Might be good to add a piece of tape or something similar to the bottom of the display for future builds? I'll be doing that for mine. Picture attached for reference. 
![HASPone-LightBleed](https://user-images.githubusercontent.com/51146434/124695108-30d3f000-dea8-11eb-8f4a-0a46003524ca.jpg)

**Comment by IIAIronWolf** [comment]
I know the pic isn't the best quality, but at the bottom of the display, the yellow and white light is visible through the 3D printed enclosure.

**Comment by aderusha** [comment]
The quality of filament you use will play a role here.  I'm now using [Overture White PETG from Amazon](https://amzn.to/3hN1I5X) as it's reasonably priced and the white is pretty opaque.  I've tested several brands and for me this has been the best solution in terms of opacity.  Another key is to make sure you have enough bottom layers on your print so that area is 100% filled.

As you noted, this is also something you might be able to fix up with a piece of tape!

I've moved this over to the discussions area as it isn't really an issue that needs to be tracked.  Thanks for the input @IIAIronWolf !
